src/bindings/java/org/simgrid/msg/Msg.java

```diff
@@ -41,6 +41,7 @@ private Msg() {
 	/** Tell the kernel that you want to use the energy plugin */
 	public static final native void energyInit();
 	public static final native void fileSystemInit();
+	public static final native void loadInit();
```

Comment:
Please add the missing javadoc. And while you're at it, you could add it to the function right before ;)

Reply:
yes of course.